clients/scheduler/src/components/time-slot-select/index.tsx

```diff
@@ -9,9 +9,14 @@ export const TimeSlotSelect = () => {
   const [providerTimeSlots, setProviderTimeSlots] = useState<ParsedSlotsType[]>([])
 
   const addTimeSlots = useCallback((newProviderTimeSlots: ParsedSlotsType[]) => {
+    const earliestAvailable = new Date();
+    //appointments within one hour can not be booked
+    earliestAvailable.setTime(earliestAvailable.getTime() + (60*60*1000));
```

Comment:
+1 for parameterizing the window, it will help keep our changes more applicable to other potential users of the embed when we push our changes back upstream.